addons/test_mail_full/tests/test_rating.py

```diff
@@ -56,46 +56,35 @@ def test_rating_rating_apply(self):
 
         # apply a rate as note (first click)
         with self.mock_mail_gateway(mail_unlink_sent=False), self.mock_mail_app():
-            record_rating.rating_apply(5, token=access_token, feedback='Top Feedback', subtype_xmlid='mail.mt_note')
-        message = record_rating.message_ids[0]
+            self.url_open(f"/rate/{access_token}/5")
```

Comment:
Isn't it possible to left current tests are they are, and instead add a new httpcase test ? It seems we could have two tests, one for main methods (rating_apply), one for the UX behavior (routes).

Reply:
Actually we plan to change that flow in a near future (the tests were done to prepare the improvements). There will be some changes afterwards.